### PR TITLE
Lint code and doc

### PR DESCRIPTION
**Pull request by bersace**
Fix the output of https://pypi.python.org/pypi/django-context-logging/1.1.0

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -1,5 +1,16 @@
...
+[testenv:lint]
+basepython = python2.7
+skipsdist = True
+skip_instal = True
+deps =
+    flake8
+    docutils
+commands =
+    flake8 --verbose src/
+    rst2html.py --strict README.rst /dev/null
```
Comment (bersace) [inline tox.ini]: If this command fail, pypi will output the rst as plain text, which is ugly.

**Comment by bersace** [comment]
Green :-)

**Comment by lorehov** [comment]
Yeah excellent :)

**Comment by lorehov** [comment]
By the way I have add travis badge and bump version a little.

**Pull request merged**